conbench/templates/benchmark-entity.html

```diff
@@ -165,6 +165,22 @@
           <a href="{{ run.commit.repository }}">{{ run.commit.display_repository }}</a>
         </div>
       </li>
+      {% if run.commit.branch %}
+      <li class="list-group-item" style="overflow-y: auto;">
+        <b>branch</b>
+        <div align="right" style="display:inline-block; float: right;">
+          {{ run.commit.branch }}
+        </div>
+      </li>
+      {% endif %}
+      {% if run.commit.fork_point_sha %}
+      <li class="list-group-item" style="overflow-y: auto;">
+        <b>forked from default at</b>
+        <div align="right" style="display:inline-block; float: right;">
+          {{ run.commit.fork_point_sha }}
+        </div>
+      </li>
+      {% endif %}
```

Comment:
I was back-and-forth on including the fork point SHA, but it was nice for testing so I left it in.

In the future when we auto-backfill all default-branch commits, each Commit entity should have a relationship to its fork-point entity, so it'll be easier to turn this SHA into a nice actual link to the commit.

<img width="576" alt="image" src="https://user-images.githubusercontent.com/16600275/199046069-90504a43-2092-4404-a533-4bc6611f43c2.png">